pkg/storaged/btrfs/subvolume.jsx

```diff
@@ -180,6 +181,149 @@ function subvolume_create(volume, subvol, parent_dir) {
     });
 }
 
+async function snapshot_create(volume, subvol, subvolume_path) {
+    const localstorage_key = "storage:snapshot-locations";
+    console.log(volume, subvol, subvolume_path);
+    const action_variants = [
+        { tag: null, Title: _("Create snapshot") },
+    ];
+
+    const get_local_storage_snapshots_locs = () => {
+        const localstorage_snapshot_data = localStorage.getItem(localstorage_key);
```

Comment:
What about using the location of the most recently made snapshot? By looking at the list of snapshots of this subvolume. That makes this independent from browser storage.